presto-main/src/test/resources/catalog.json

```diff
@@ -0,0 +1,40 @@
+{
+  "catalogs": [
+    {
+      "user": "admin",
+      "catalog": ".*",
+      "allow": true
+    },
+    {
+      "catalog": "secret",
+      "allow": false
+    },
+    {
+      "user": ".*",
+      "catalog": "open-to-all",
+      "allow": true
+    },
+    {
+      "catalog": "all-allowed",
+      "allow": true
+    },
+    {
+      "user": "alice",
+      "catalog": "alice-catalog",
+      "allow": true
+    },
+    {
+      "user": "bob",
```

Comment:
this rule isn't explicitly necessary, right? since bob by default has no permissions on alice-catalog.